docs/synthetics/ci-cd-integration.md

```diff
@@ -0,0 +1,300 @@
+* `<locationId>` - List of locations to run the monitor from. If not specified, the monitor will be run from all locations specified in the monitor configuration. The supported locations are:
+
+| Location ID  | Location |
+|---|---|
+| us-east-1      | N. Virginia, USA   |
+| eu-west-1      | Ireland            |
+| ap-south-1     | Mumbai, India      |
+| ap-southeast-1 | Singapore          |
+| ap-southeast-2 | Sydney, Australia  |
+| eu-central-1   | Frankfurt, Germany |
+| sa-east-1      | Sao Paulo, Brazil  |
+| us-west-1      | N. California, USA |
+
+
+**Example Request**
+
+The below API triggers runs for monitors with id `276` and `335` belonging to App with id `12345` from locations `N.Virginia, USA`, `Mumbai, India` and `N.Virginia, USA`, `Ireland, Europe` respectively.
+
+```sh
+curl --request POST \
+  --url https://apps.sematext.com/synthetics-api/api/v3/apps/12345/monitors/runs \
+  --header 'Authorization: apiKey 1d7a2d6b-xxxx-xxxx-xxxx-10f83c5c8da7' \
+  --header 'Content-Type: application/json' \
+  --data '[
+    {
+        "monitorId": 276,
+        "regions": ["us-east-1","ap-south-1"]
+    },
+    {
+        "monitorId": 335,
+        "regions": ["us-east-1","eu-west-1"]
+    }
+]'
+```
+
+### Customize Request Configuration
+
+There are cases where you might want to customize request parameters depending on the environment. For example, the deployment URL for running the monitor in PR env or a different HTTP header for the staging environment. You can pass these custom configurations as part of run monitor API data. When the custom values are passed the configured values for scheduled runs will be overridden with the custom values.
+
+For HTTP monitors, the following fields can be customized:
+
+* URL
+* Request Headers
+* Request Cookies
+* Request Params
+* Request Body
+
+Below is an example, where we are override the HTTP configuration parameters:
+
+```sh
+curl --request POST \
+  --url https://apps.sematext.com/synthetics-api/api/v3/apps/12345/monitors/runs \
+  --header 'Authorization: apiKey 1d7a2d6b-xxxx-xxxx-xxxx-10f83c5c8da7' \
+  --header 'Content-Type: application/json' \
+  --data '[
+    {
+        "monitorId": 276,
+        "url": "https://pr-2589.app.acme.com",
+        "headers": [
+            {
+                "name": "Authorization",
+                "value": "pr-env-key"
+            }
+        ]
+    }
+]'
+```
+
+For Browser monitors, the URL of the website can be customized.
+
+For Browser monitors with a script, you can pass custom parameters as variables, that could be referenced in the script.
+
+### Customize Output Format
+
+By default the API output will be in JSON format. While invoking the API in build pipelines, it would be useful to display the output in a table format, so that the output could be easily interpreted. To get the output in table format, set the `Accept` header to `text/plain`. Below is an example request along with the output:
```

Comment:
it would be useful => it is useful
or
it would be useful => it might be useful

Reply:
done